_sub/compute/helm-kube-prometheus-stack/values/node-exporter.yaml

```diff
@@ -1,9 +1,9 @@
-prometheus-node-exporter:
-  resources:
-    requests:
-      cpu: 20m
-      memory: 50Mi
-  priorityClassName: ${prometheus_node_exporter_priorityclass}
-  prometheus:
-    monitor:
-      enabled: true
+# prometheus-node-exporter:
```

Comment:
I guess these comments are temporarily and will be removed in a later PR?

Reply:
I am keeping them here for reference, and they will removed when we remove this module